oletools/util/log_helper/_root_logger_wrapper.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+
+def is_logging_initialized():
+    """
+    We use the same strategy as the logging module when checking if
+    the logging was initialized - look for handlers in the root logger
+    """
+    return len(logging.root.handlers) > 0
+
+
+def set_formatter(fmt):
```

Comment:
Since every logger will propagate to the root logger when logging, we only need to set the formatter of the root logger, which will be applied to every logger we created.

Note that this function is only called by [`enable_logging()`](https://github.com/christian-intra2net/oletools/blob/7cb86312f0a9386926b56e9c8f5578090083edfe/oletools/util/log_helper/log_helper.py#L110) which in turn is only called in the main() function of modules. This means that it won't affect scripts importing oletools.